<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditStaffBase</class>
 <widget class="QDialog" name="EditStaffBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MuseScore: Edit Staff/Part Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="3">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="musescore.qrc">:/data/bg1.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Staff Properties</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="small">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Small</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="invisible">
        <property name="text">
         <string>Invisible</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>79</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="staffType"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>79</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Part Properties</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0,0,0,1">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item row="2" column="1">
         <widget class="QLineEdit" name="instrumentName">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Instrument:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="changeInstrument">
          <property name="text">
           <string>Change Instrument...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Part Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="partName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Long Instrument Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Short Instrument Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTextEdit" name="longName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTextEdit" name="shortName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="editLongName">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="editShortName">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Usable Pitch Range:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Amateur:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="minPitchA">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="minPitchASelect">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="musescore.qrc">
            <normaloff>:/data/table.svg</normaloff>:/data/table.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxPitchA">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="maxPitchASelect">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="musescore.qrc">
            <normaloff>:/data/table.svg</normaloff>:/data/table.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Professional:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="minPitchP">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="minPitchPSelect">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="musescore.qrc">
            <normaloff>:/data/table.svg</normaloff>:/data/table.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxPitchP">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="maxPitchPSelect">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="musescore.qrc">
            <normaloff>:/data/table.svg</normaloff>:/data/table.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Play Transposition:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="iList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>0 - Perfect Unison</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 - Augmented Unison</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0 - Diminished Second</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 - Minor Second</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 - Major Second</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 - Augmented Second</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 - Diminished Third</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 - Minor Third</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4 - Major Third</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5 - Augmented Third</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4 - Diminished Fourth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5 - Perfect Fourth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6 - Augmented Fourth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6 - Diminished Fifth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7 - Perfect Fifth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8 - Augmented Fifth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7 - Diminished Sixth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8 - Minor Sixth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9 - Major Sixth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10 - Augmented Sixth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9 - Diminished Seventh</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10 - Minor Seventh</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11 - Major Seventh</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12 - Augmented Seventh</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11 - Diminished Octave</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12 - Perfect Octave</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>+octave</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="octave">
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="up">
          <property name="text">
           <string>up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="down">
          <property name="text">
           <string>down</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Number of strings:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numOfStrings">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editStringData">
          <property name="text">
           <string>Edit string data...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="musescore.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditStaffBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditStaffBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
